### DISCUSSION TITLE
Delayed Payouts and Stuck Transactions

### DISCUSSION THREAD

**Discussion by zuhaibdct** [btcpayserver/btcpayserver#6035] [Technical Support]
Payouts are experiencing significant delays and appear to be stuck in an infinite block. A specific transaction (ID: 4afc962aa152439e75316193c13491899ad73697de12f4d3f38fdccc51131e3c) has been pending for an extended period without confirmation.

**Comment by mp21-org** [comment]
according to https://mempool.space/tx/4afc962aa152439e75316193c13491899ad73697de12f4d3f38fdccc51131e3c you have 5 unconfirmed inputs in your transaction. So your configured fees are also used to bump them which lowers your effective fee rate to 23,9 sat / vB which is currently not enough for a quick confirmation.

**Reply by zuhaibdct** [reply]
![image](https://github.com/btcpayserver/btcpayserver/assets/108730407/a93537a5-845d-466c-a3d2-ace932574ace)
In the payout processor, I am using fee block target =1.
Is this correct for quick confirmation and is there any way to quickly confirm the above transaction?

**Reply by dennisreimann** [reply]
>Is this correct for quick confirmation

Yes, but you can only spend confirmed inputs. 

> is there any way to quickly confirm the above transaction?
 
If your payout transaction contains outputs, which haven't been confirmed yet, you'd need to CPFP fee bump.

**Reply by mp21-org** [reply]
I'd somehow expect the fee estimator here to use the effective fee rate, looking at the fees needed to bump the unconfirmed inputs as well.

Or a setting in the payout processor to only use confirmed inputs (it seemingly didn't do it in the transaction provided by @zuhaibdct ), don't see it here:

![grafik](https://github.com/btcpayserver/btcpayserver/assets/1374810/07cb7f67-2dc4-4f4c-a809-30ec56108ef3)

**Reply by zuhaibdct** [reply]
I agree with @petzsch it will help a lot
